packages/lesswrong/components/tagging/TagEditsTimeBlock.tsx

```diff
@@ -33,9 +33,13 @@ const TagEditsTimeBlock = ({before, after, reportEmpty, classes}: {
         lastCommentedAt
         added
         removed
+        users {
+          ...UsersMinimumInfo
+        }
       }
```

Comment:
It's really hard to tell from reading this that this GraphQL query is getting multiple results from `TagUpdatesInTimeBlock`. (That's a commentary on the query language, not on this PR).